content/modeling/modeling_1.md

```diff
@@ -0,0 +1,95 @@
++++
+title= "Set-up and run core-web-vitals module"
+weight = 3
+post = ""
++++
+
+
+#### **Step 1:** Enable the optional Core web vitals module
+As the [advanced-analytics-for-web accelerator](https://docs.snowplow.io/accelerators/web/) is a prerequisite, it is assumed that you already have a dbt project set-up to process basic web events.
+
+To enable the optional `core web vitals` module, you must add the following code snippet to your dbt_project.yml file:
+
+
+```yml
+models:
+  snowplow_web:
+    snowplow__enable_cwv: true
+```
+
+#### **Step 2:** Change optional parameters
+
+By default, following industry standard practice, the core web vitals are evaluated against the 75 percentile value of all pageviews. You can use the `snowplow__cwv_percentile` variable to overwrite this logic.
+
+The measurements are executed in a drop and recompute fashion to fit BI tool purposes and would always show the last 28 days data by default which can be modified through a variable in the model (`snowplow__cwv_days_to_measure`).
+
+To change either of these default values you can modify the relevant variable in your dbt_project.yml file like so:
+
+```yml
+models:
+  snowplow_web:
+    snowplow__enable_cwv: true
+    snowplow__cwv_days_to_measure: 28
+    snowplow__cwv_percentile: 75
+```
+
+### **Step 3:** Override the module specific macros
+
```

Comment:
Maybe make it clear this step is optional in the header?

Reply:
They would need to do one of them at least

Reply:
the URL grouping one I mean